### PR TITLE
OTT-1377: Print stats to be sent to stats-server

### PR DESCRIPTION
**Pull request by ashishshinde-pubm**
Code changes to print the stats metrics if loglevel is debug.

### CODE REVIEW

File: modules/pubmatic/openwrap/metrics/stats/client.go [hunk 1]
```diff
@@ -150,6 +150,11 @@ func (sc *Client) publishStatsToServer(statMap map[string]int) int {
 
 		if err == nil && code == http.StatusOK {
 			glog.Infof("[stats_success] retry:[%d] nstats:[%d] time:[%v]", retry, len(statMap), elapsedTime)
+			if glog.V(3) {
+				if statJson, err := json.Marshal(statMap); err == nil && len(statJson) > 0 {
+					glog.Infof("[stats_success] stats to be sent to stats-server :%+v", string(statJson))
+				}
+			}
```
Comment (pm-viral-vala) [inline modules/pubmatic/openwrap/metrics/stats/client.go:157]: can we have this irrespective of stats_success or stats_failure?
Incase of failure we won't be able to see any stats

**Review by pm-viral-vala — CHANGES_REQUESTED** [review]

File: modules/pubmatic/openwrap/metrics/stats/client.go [hunk 2]
```diff
@@ -123,13 +123,14 @@ func (sc *Client) prepareStatsForPublishing() {
...
 
-	sb, err := json.Marshal(statMap)
+	statJson, err := json.Marshal(statMap)
 	if err != nil {
 		glog.Errorf("[stats_fail] Json unmarshal fail: %v", err)
 		return statusSetupFail
 	}
 
-	req, err := http.NewRequest(http.MethodPost, sc.endpoint, bytes.NewBuffer(sb))
+	glog.V(3).Infof("stats to be sent to stats-server :%+v", string(statJson))
```
Comment (pm-viral-vala) [inline modules/pubmatic/openwrap/metrics/stats/client.go:132]: please use %s and remove string conversion

Comment (pm-viral-vala) [reply modules/pubmatic/openwrap/metrics/stats/client.go:132]: please log it with some prefix 
```glog.V(3).Infof("[stats] nstats:[%d] data:[%s]",len(statMap). statJson)```

**Review by pm-viral-vala — CHANGES_REQUESTED** [review]

File: modules/pubmatic/openwrap/metrics/stats/client.go (hunk 2 above)
Comment (ashishshinde-pubm) [reply modules/pubmatic/openwrap/metrics/stats/client.go]: done

**Review by PubMatic-OpenWrap — APPROVED** [review]

**Review by pm-viral-vala — APPROVED** [review]

**Pull request merged**